### ISSUE TITLE
[ Suggestion ] Add config to remove all of one parts type of one element

### ISSUE THREAD

**Issue by HurtsAsHell** [MJRLegends/TinkersExtras#7]
### I'm not very good in English so I'm sorry, I'll try to make myself understood

In this mod there is the possibility to remove a part of a material, such as: *pick_head:stone*
Do something like this for a type of part such as : *head:stone*;
This disables all the head of the stone material ( pick_head:stone, sword_blade:stone, shovel_head:stone etc)

I do not know how your code is made, but I think it should not be possible to do, and it would really help a lot

**Sorry again for not perfect translations**

**Comment by HurtsAsHell** [comment]
yes but in this way I will have to repeat this procedure for all the heads (axe, pickaxe, sword etc.)

I asked to create, an array for example, that contains all the parts that the tinker considers HEAD and with a check to eliminate those of a certain material

**Comment by HurtsAsHell** [comment]
yes but in this way I will have to repeat this procedure for all the heads (axe, pickaxe, sword etc.)

I asked to create, an array for example, that contains all the parts that the tinker considers HEAD and with a check to eliminate those of a certain material

@MJRLegends

**Comment by MJRLegends** [comment]
Maybe will look in to it. But it might be more hassle than its worth.
